src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -163,7 +171,7 @@ impl CloudwatchLogsSinkConfig {
     fn create_client(&self) -> crate::Result<CloudWatchLogsClient> {
         let region = (&self.region).try_into()?;
 
-        let client = rusoto::client()?;
+        let client = rusoto::client(&self.proxy)?;
```

Comment:
It seems like you missed this one when fixing the AWS components to take the global proxy settings into account.